librz/core/core_private.h

```diff
@@ -119,7 +119,7 @@ RZ_IPI RzCmdStatus rz_core_binldr_plugin_print(const RzBinLdrPlugin *ld, RzCmdSt
 /* cdebug.c */
 RZ_IPI bool rz_core_debug_reg_set(RzCore *core, const char *regname, ut64 val, const char *strval);
 RZ_IPI bool rz_core_debug_reg_list(RzCore *core, int type, int size, PJ *pj, int rad, const char *use_color);
-RZ_IPI void rz_core_debug_regs2flags(RzCore *core, int bits);
```

Comment:
Since you touched `rz_core_debug_regs2flags`, I hope you can add a oneliner Doxygen doc for the API.